Dockerfile

```diff
@@ -24,18 +24,13 @@ RUN apt-get update && \
     curl -L https://github.com/aelsabbahy/goss/releases/download/${GOSS_VERSION}/goss-linux-amd64 -o /usr/local/bin/goss && \
     chmod +x /usr/local/bin/goss && \
     apt-get remove --purge -yq \
-        curl \
-    && \
-    apt-get autoclean -y && \
-    apt-get autoremove -y && \
-    rm -rf /var/lib/{cache,log}/ && \
-    rm -rf /var/lib/apt/lists/*.lz4 && \
-    rm -rf /tmp/* /var/tmp/*
+      curl
 
 # Overlay the root filesystem from this repo
 COPY ./container/root /
 
-RUN goss -g goss.base.yaml validate
+RUN goss -g goss.base.yaml validate && \
+    /bin/bash /clean.sh
```

Comment:
Good to know. When running clean.sh as part of the installation step docker image is using 138.8 MB.
I have another question: When i am running clean.sh in different docker layer the size of docker image is 192 MB. When i ssh into that container and check the disk space by running the command `du -skh /` it shows 124 MB also those files (deleted by clean.sh) are not present. Why is docker image showing 192 MB and not 138.8 MB?